.gitignore

```diff
@@ -43,3 +43,9 @@ lib/*
 !strongbox-security/strongbox-authentication-registry/src/test/resources/webapp/WEB-INF/lib/strongbox-example-authentication-provider-1.0-SNAPSHOT.jar
 !strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/resources/artifacts/properties-injector-1.7.jar
 *.sh
+.idea/
+strongbox-web-core/db/
```

Comment:
This is not right. These should all be under the `target/strongbox-vault` and it's handled via properties when starting up the server.